swagger-spec/taxonomies/list.yaml

```diff
@@ -1 +1,175 @@
 # /taxonomies/
+get:
+  summary: List all taxonomies
+  description: >-
+
+    A paginated list of all [PLOS taxonomies of subjects](http://journals.plos.org/plosone/browse/).
```

Comment:
We're actually using the bepress disciplines taxonomy now, not PLOS'. Can you reword this and include [this link](https://www.bepress.com/wp-content/uploads/2016/12/Digital-Commons-Disciplines-taxonomy-2017-01.pdf) instead?